doc/architecture/decisions/0036-use-audited-for-budget-revisions.md

```diff
@@ -0,0 +1,53 @@
+# 36. Use Audited for Budget Revisions
+
+Date: 2023-03-15
+
+## Status
+
+Accepted
+
+## Context
+
+Budgets can go through a number of revisions; they can be updated over time as
+more or less funding comes in, or the scope of the activity changes.
+
+We want to be able to keep a history of these revisions. They will be displayed
+on a new revisions page and exported as "current" and "original" budgets.
+
+### Audited
+
+The [Audited Gem](https://github.com/collectiveidea/audited) provides us with
```

Comment:
`gem` (same on line 50)

Reply:
Hmm, I wouldn't given it's not a proper noun, but style choices vary!

Reply:
RubyGems agree with you! Is this then a "common noun" 🤔

Reply:
I'd say so, given it's not referring to a specific individual thing by name (e.g. Audited) but rather a type of thing, of which there are many instances